protos/bottle/catalog/v1/addon.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package bottle.catalog.v1;
+
+message Addon {
+  enum AddonCategory {
+    ADDON_CATEGORY_UNSPECIFIED = 0;
+    ADDON_CATEGORY_RUSH_ASSEMBLY = 1;
```

Comment:
I didn't include any pricing logic here because that's just going to be set on the order line item. No linking it to other products on the order because it's unneeded right now. I also expect this to include warranty down the line.